conmon/conmon.c

```diff
@@ -582,18 +694,28 @@ int main(int argc, char *argv[])
 					pipe = STDOUT_PIPE;
 				else if (masterfd == masterfd_stderr)
 					pipe = STDERR_PIPE;
+				else if (oom_handling_enabled && masterfd == efd) {
+					if (read(efd, &oom_event, sizeof(uint64_t)) != sizeof(uint64_t))
+						nwarn("read eventfd");
```

Comment:
A little more verbiage specifying the reason for the warning would be good here.  "Reading of eventfd failed as it was not the expected size." perhaps?